pkg/operator/ceph/object/bucket/controller.go

```diff
@@ -53,6 +54,10 @@ type ReconcileBucket struct {
 // Add creates a new Ceph CSI Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager, context *clusterd.Context, opManagerContext context.Context, opConfig opcontroller.OperatorConfig) error {
+	if os.Getenv("SKIP_OBJECT_BUCKET_WATCH") == "true" {
```

Comment:
How about this variable name? 
```suggestion
	if os.Getenv("REQUIRE_OBC_CRDS") == "false" {
```